cypress/e2e/rails_examples/using_factory_bot.cy.js

```diff
@@ -0,0 +1,21 @@
+describe('Rails using factory bot examples', function() {
+  beforeEach(() => {
+    cy.app('clean') // have a look at e2e/app_commands/clean.rb
+  })
+
+  afterEach(() => {
+    cy.app('clean')
+  })
+
+  it('using single factory bot', function() {
+    cy.appFactories([
+      ['create', 'vm_vmware', {name: 'test_vm'} ]
+    ])
+    cy.login();
+    cy.menu('Compute', 'Infrastructure', 'Virtual Machines');
+    let main_div = cy.get('div#main_div')
+    // clean should of removed these from other tests
+    main_div.should('not.contain', 'test_host');
+    main_div.contains('test_vm');
```

Comment:
This is the only generated file in rails_examples that I modified to use our application.  It runs fine and can be run over and over again, so the clean is working.